modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -7,9 +7,11 @@
       <component
         v-if="fieldState[field.name]"
         v-model="fieldState[field.name]"
-        :is="fieldComponentMap[field.type]" :field="fields[field.name].field"
+        :is="fieldComponentMap[field.type]"
+        :field="fields[field.name].field"
         :status="fields[field.name].status"
         :modifiers="fields[field.name].modifiers"
+        :doc-id="fieldState._id && fieldState._id.data"
```

Comment:
Only `AposInputSlug` needs this. Unfortunately, every field in the schema will have this additional prop.

Reply:
True, but I think it is OK because the doc id doesn't change, so it doesn't make reactivity less efficient.

I'm not sure why _id is treated as a field though since it really isn't one. It should probably be an independent prop passed to AposSchema, but fixing that is out of scope for this PR 😆

Reply:
Every doc doesn't need to accept it unless this is a feature we actually want to make available. If that's not the case, which I don't think it is yet, those props should only be on this component, not the mixin.

Reply:
@boutell It's treated as a field because of the recent work in the media library. To know if the media library editor needs to replace the doc data it needed a unique value to check. https://github.com/apostrophecms/apostrophe/pull/2393#pullrequestreview-490535691